docs/products/cassandra/get-started.md

```diff
@@ -4,34 +4,418 @@ sidebar_label: Get started
+1. Update the `aiven_cassandra` resource in the `cassandra-sample.tf` file:
+
+   - Add `service_log = true` and `termination_protection = true`.
+   - Update `maintenance_window_dow = "sunday"` and `maintenance_window_time = "22:00:00"`.
+
+    ```hcl
+    variable "aiven_token" {
+      type = string
+    }
+
+    variable "aiven_project_name" {
+      type = string
+    }
+
+    terraform {
+      required_providers {
+        aiven = {
+          source  = "aiven/aiven"
+          version = ">=4.0.0, <5.0.0"
+        }
+      }
+    }
+
+    provider "aiven" {
+      api_token = var.aiven_token
+    }
+
+    resource "aiven_cassandra" "example_cassandra" {
+      project                 = var.aiven_project_name
+      cloud_name              = "google-europe-west1"
+      plan                    = "startup-4"
+      service_name            = "cassandra-sample"
+      maintenance_window_dow  = "sunday"
+      maintenance_window_time = "22:00:00"
+      termination_protection  = true
+
+      cassandra_user_config {
+        migrate_sstableloader = true
+        service_log = true
+      }
+    }
+
+    output "cassandra_service_host" {
+      value = aiven_cassandra.example_cassandra.service_host
+    }
+
+    output "cassandra_service_port" {
+      value = aiven_cassandra.example_cassandra.service_port
+    }
+
+    output "cassandra_service_username" {
+    value = aiven_cassandra.example_cassandra.service_username
+    }
+
+    output "cassandra_service_password" {
+      value     = aiven_cassandra.example_cassandra.service_password
+      sensitive = true
+    }
+    ```
+
+1. Run `terraform plan` > `terraform apply`.
+
+</TabItem>
+<TabItem value="3" label="K8s">
+1. Update file `cassandra-sample.yaml`:
+
+   - Add `service_log: true` and `terminationProtection: true`.
+   - Update `maintenanceWindowDow: sunday` and `maintenanceWindowTime: 22:00:00`.
+
+   ```yaml
+   apiVersion: aiven.io/v1alpha1
+   kind: Cassandra
+   metadata:
+     name: cassandra-sample
+   spec:
+     authSecretRef:
+       name: aiven-token
+       key: token
+
+     connInfoSecretTarget:
+       name: cassandra-secret
+
+     userConfig:
+       migrate_sstableloader: true
+       service_log: true
+
+     project: PROJECT_NAME
+     cloudName: google-europe-west1
+     plan: startup-4
+
+     maintenanceWindowDow: sunday
+     maintenanceWindowTime: 22:00:00
+     terminationProtection: true
+   ```
+
+1. Update the service by applying the configuration:
+
+   ```shell
+   kubectl apply -f cassandra-sample.yaml
+   ```
+
+1. Review the resource you updated with the following command:
+
+   ```shell
+   kubectl describe cassandra.aiven.io cassandra-sample
+   ```
+
+The resource can stay in the `REBUILDING` state for a couple of minutes. Once the state
+changes to `RUNNING`, you are ready to access it.
+
+See the available configuration options in
+[Aiven Operator for Kubernetes®: Cassandra](https://aiven.github.io/aiven-operator/api-reference/cassandra.html).
```

Comment:
The TF equivalent seems to be an admonition at the top of the tab. Should we align the tabs on one approach/pattern?